jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AdminResource.java

```diff
@@ -55,6 +64,8 @@
 import io.swagger.annotations.ApiOperation;
 import io.swagger.annotations.ApiResponse;
 import io.swagger.annotations.ApiResponses;
+import net.sf.ehcache.CacheManager;
```

Comment:
It works today because ehcache is a transitive dependency of killbill-util, already in the jaxrs pom.xml. But better be explicit about it and add the `net.sf.ehcache:ehcache` dependency.